ocean-client/src/utils/store.ts

```diff
@@ -60,6 +61,8 @@ export class StoredSettings {
   mainCollateralAsset: string = 'DFI'
   stableCoinArbBatchSize: number = -1
   reinvestThreshold: number | undefined
+  sendThreshold: number | undefined
+  sendAll: boolean = false
```

Comment:
probably time to refactor this, so that every program-store has its own stored settings. right now this is becoming a mix of all of them.